demo/src/features/debug3d/extract.rs

```diff
@@ -34,7 +31,7 @@ impl ExtractJob for Debug3dExtractJob {
 
         let debug3d_material = &extract_context
             .render_resources
-            .fetch::<Debug3DStaticResources>()
+            .fetch::<StaticResources>()
```

Comment:
Would keep the Debug3D prefix

Reply:
If `StaticResources` are intended to be public, then I agree and I'll add the prefixes back. 
If they are not intended to be public, this discussion should resolve according to https://github.com/aclysma/rafx/pull/122#discussion_r610704300.

Reply:
Per discussion offline, will add the prefix & remove the suffix.